spring-integration-core/src/main/java/org/springframework/integration/graph/MicrometerNodeEnhancer.java

```diff
@@ -119,8 +123,12 @@ private <T extends IntegrationNode> Timer obtainTimer(T node, String type, boole
 		}
 	}
 
-	@Nullable
-	private <T extends IntegrationNode> Timer observationTimer(T node, String type, boolean success) {
+	private RequiredSearch getRegistry(String name) {
```

Comment:
We don't need this method any more after that `NullAway.Init`.
And therefore everything else.